packages/rocket-static-sites-aws-infrastructure/src/static-website-stack.ts

```diff
@@ -0,0 +1,76 @@
+import { CfnOutput, RemovalPolicy, Stack } from '@aws-cdk/core'
+import { Bucket } from '@aws-cdk/aws-s3'
+import { BucketDeployment, Source } from '@aws-cdk/aws-s3-deployment'
+import {
+  CloudFrontWebDistribution,
+  OriginAccessIdentity,
+  ViewerCertificate,
+  ViewerProtocolPolicy,
+} from '@aws-cdk/aws-cloudfront'
+import { existsSync } from 'fs'
+import { RocketUtils } from '@boostercloud/framework-provider-aws-infrastructure'
+
+export type AWSStaticSiteParams = {
+  localDistPath?: string
```

Comment:
Where does the `localDist` part come from? 

From the code, this seems to be the root folder of the static file. What do you think about calling it `rootPath`?